src/service/httpFetch.js

```diff
@@ -21,6 +21,17 @@ export function getUserDetails() {
   });
 }
 
+
+export function getFeeds(username) {
```

Comment:
Remove code duplication here. Make axios configuration common.

Reply:
if i create a common config object and then assign to axios, it faces an issue. 
when i sign in , the API call to fetch the user data gives 401 error bcz the common config is not refreshed and still gets the stale value of token..